code/game/objects/items/stunbaton.dm

```diff
@@ -241,9 +241,18 @@
 
 	if(ishuman(L))
 		var/mob/living/carbon/human/H = L
+		var/datum/mind/M = H.mind
+		if(M?.assigned_role == "Assistant" && user.a_intent == INTENT_HARM)
+			var/amount_given = 1
```

Comment:
this will runtime since amount_given is defined in the if statement

Reply:
the clown statement is in the same if statement...